webgpu.h

```diff
@@ -3625,6 +3625,12 @@ typedef struct WGPUBindGroupLayoutEntry {
      * The `INIT` macro sets this to `0`.
      */
     uint32_t binding;
+    /**
+     * The number of bindings (size of the binding array) defined by this entry.
+     *
+     * The `INIT` macro sets this to `1`.
+     */
+    uint32_t bindingCount;
```

Comment:
Could we have this set to 0 meaning not a binding array? The JS spec can decide what 0 means (vs undefined) separately.

Also I don't think the rename is an improvement, see gpuweb comment.